injectproxy/routes.go

```diff
@@ -118,27 +120,65 @@ func (r *routes) noop(w http.ResponseWriter, req *http.Request) {
 }
 
 func (r *routes) query(w http.ResponseWriter, req *http.Request) {
-	expr, err := parser.ParseExpr(req.FormValue("query"))
+	// The `query` can come in the URL query string and/or the POST body.
+	// For this reason, we need to try to enforcing in both places.
```

Comment:
```suggestion
	// , so enforce in label in both places.
```

Reply:
`Enforce in label` is not syntactically correct.i would prefer as is

Reply:
"try" is also weird though. We don't try. We do or fail.

Reply:
Not a biggy.